jwst/extract_2d/extract_2d.py

```diff
@@ -62,9 +65,17 @@ def extract2d(input_model, slit_name=None, apply_wavecorr=False,
                                               extract_height=extract_height,
                                               extract_orders=extract_orders)
         else:
+            if exp_type is 'NRC_WFSS':
+                use_fits_wcs = True
```

Comment:
The hardwire is just another parameter I added, `extract_2d` currently looks to see if the instrument is nircam and sends `use_fits_wcs` down through to the box calculating code which will then use the fits wcs only to translate the catalog source location and bounding box corners to the pointing position of the grism image. gwcs is used through the grism. Because the team only has simulations right now, and the distortion file for image mode (which wfss mode uses for this specific calculation) is off, source_catalog is using fits wcs,  and we are still debugging the sim creation, having this option in place for scsb and the science team to use I think is the best option to help disentangle issues...without writing lots of helper scripts and always having to change the pipeline results. As soon as the updated distortion reference is sent, the translation should be closer between fits wcs and gwcs and we can take this out or set to false.